data_steward/validation/main.py

```diff
@@ -633,36 +627,6 @@ def extract_date_from_rdr_dataset_id(rdr_dataset_id):
     raise ValueError(f"{rdr_dataset_id} is not a valid rdr_dataset_id")
 
 
-def get_hpo_missing_pii_query(hpo_id):
-    """
-    Query to retrieve counts of drug classes in an HPO site's drug_exposure table
-
-    :param hpo_id: identifies the HPO site
-    :return: the query
-    """
-    person_table_id = bq_utils.get_table_id(hpo_id, common.PERSON)
-    pii_name_table_id = bq_utils.get_table_id(hpo_id, common.PII_NAME)
-    pii_wildcard = bq_utils.get_table_id(hpo_id, common.PII_WILDCARD)
-    participant_match_table_id = bq_utils.get_table_id(hpo_id,
-                                                       common.PARTICIPANT_MATCH)
-    rdr_dataset_id = bq_utils.get_rdr_dataset_id()
-    rdr_date = extract_date_from_rdr_dataset_id(rdr_dataset_id)
-    ehr_no_rdr_with_date = consts.EHR_NO_RDR.format(date=rdr_date)
-    rdr_person_table_id = common.PERSON
-    return render_query(
-        consts.MISSING_PII_QUERY,
-        person_table_id=person_table_id,
-        rdr_dataset_id=rdr_dataset_id,
-        rdr_person_table_id=rdr_person_table_id,
-        ehr_no_pii=consts.EHR_NO_PII,
-        ehr_no_rdr=ehr_no_rdr_with_date,
```

Comment:
@mark-velez From the scope of this ticket, I think only this `ehr_no_rdr`  part is to be deleted. `missing_pii` needs to stay.